common/constants.gradle

```diff
@@ -0,0 +1,14 @@
+project.ext.projectName = "firebase-jobdispatcher-android"
```

Comment:
this would arguably be tidier as

project {
  ext {
    a = b

but I don't really mind